documentation/tutorials/Demo_03_Meta_Analysis/meta_analysis.qmd

```diff
@@ -0,0 +1,278 @@
+
+## What This Notebook Does
+
+This notebook demonstrates how to:
+
+1.  Load and configure PEcAn settings for meta-analysis.
+2.  Understand the `meta.analysis` settings in `pecan.xml`.
+3.  Run the meta-analysis module using pre-generated trait data.
+4.  Analyze and visualize the posterior distributions and MCMC diagnostics.
+
+## The Scenario
+
+**Context & modeling scenario:**
+
+This demo supports the modeling scenario introduced in [Demo 01](../Demo_1_Basic_Run/run_pecan.qmd) and [Demo 02](../Demo_02_Uncertainty_Analysis/uncertainty.qmd), which simulated ecosystem carbon balance at the AmeriFlux Niwot Ridge Forest site ([US‑NR1](https://ameriflux.lbl.gov/sites/siteinfo/US-NR1)) using the SIPNET model.
+
+While those demos focused on running the ecosystem model, this notebook zooms in on the **Meta-Analysis** step. We will generate informative posterior distributions for the **temperate coniferous** Plant Functional Type (PFT) used in those simulations. These posteriors can then be used to constrain the parameters of the SIPNET model (or other models) to improve prediction accuracy.
+
+In this specific demo, we are performing a meta-analysis for the **temperate coniferous** Plant Functional Type (PFT). The goal is to estimate the probability distributions for key model parameters (e.g., SLA, leaf turnover rate) by combining:
+*   **Priors**: Existing knowledge about the parameters.
+*   **Data**: Observed trait data from the BETYdb database (pre-generated for this demo).
+
+# Prerequisites
+
+Before running this notebook, make sure you have:
+
+*   **PEcAn packages installed**:
+*   **For this demo**: You only need a subset of PEcAn packages to run the meta-analysis.
+```
+# Enable repository from pecanproject
+options(repos = c(
+  pecanproject = 'https://pecanproject.r-universe.dev',
+  CRAN = 'https://cloud.r-project.org'))
+# Install core packages for meta-analysis
+install.packages(c('PEcAn.MA', 'PEcAn.settings', 'PEcAn.utils', 'PEcAn.logger'))
+```
+       
+*   **A valid `pecan.xml` configuration file**: Start with the example at `pecan/documentation/tutorials/Demo_03_Meta_Analysis/pecan.xml`.
+*   **Pre-generated Data**: This demo relies on `trait.data.Rdata` and `prior.distns.Rdata` files which are included in the `pft/temperate.coniferous` directory.
+
+# Load PEcAn Packages
+
+First, we need to load the PEcAn R packages. These packages provide all the functions we'll use to run the workflow.
+
+```{r libraries}
+# Load necessary PEcAn packages
+# library("PEcAn.all") # Alternatively, load all packages if installed
+library("PEcAn.settings")
```

Comment:
If you switch to meta_analysis_standalone you might only need PEcAn.MA and could push the other library loads to the end when you show the module version.